ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlow.scala

```diff
@@ -38,12 +40,14 @@ object CommandTrackerFlow {
         SubmissionMat],
       createCommandCompletionSource: LedgerOffset => Source[CompletionStreamElement, NotUsed],
       startingOffset: LedgerOffset,
-      backOffDuration: FiniteDuration = 1.second): Flow[
+      maxDeduplicationTime: () => JDuration,
```

Comment:
Why does this have to be a `java.time.Duration` instead of e.g. `scala.concurrent.duration.Duration`?

Reply:
We generally use `java.time.Instant` in our codebase. `java.time.Duration` works better with it.